apollos-church-api/src/data/index.js

```diff
@@ -76,6 +88,13 @@ const data = {
   Persona,
   Person,
   BinaryFiles,
+  // postgres stuff, we include these so the schema matches the mock resolvers
```

Comment:
idea! you can import the resolvers directly and then export them as part of `noPostgres`

Reply:
schema you mean? yea I like that better.

Reply:
yah, schema.